frontend/src/services/api/__tests__/datahelper.ts

```diff
@@ -0,0 +1,35 @@
+import { inputCSS } from 'react-select/src/components/Input';
+import { Transition } from '../../../domain/transition';
+import DataHelper from '../DataHelper';
+import FakeApi from '../FakeApi';
+
+describe('Testing Data Helper Functions', () => {
+  it('transforms transitions', async () => {
+    const api = new FakeApi();
+    const transitions: Transition[] = await api.getTransitions({
+      socCode: '12345',
+    });
+    expect(DataHelper.transformNumber(NaN, 1)).toBe(NaN);
+    expect(DataHelper.transformNumber(-10, 2)).toBe(NaN);
+    expect(DataHelper.transformNumber(NaN, NaN)).toBe(NaN);
+    expect(DataHelper.transformNumber(10.01, -2)).toBe(NaN);
+    expect(DataHelper.transformNumber(0.00001, 100)).toBe(NaN);
+    expect(DataHelper.transformNumber(-0.1, 9)).toBe(NaN);
+
+    transitions.forEach(
```

Comment:
This tests that `transformNumber` works as expected and also that the fake API returns values are valid inputs to `transformNumber`. I don't think we need to test the latter, as it's using fake data. Consider updating this to hardcode inputs/outputs as you do for the NaN cases to improve readability:

`expect(transformNumber(53.234, 2)).toBe(52.23)`